client/knowledge_base_search/src/__test__/containers/SearchContainer/FeatureSelect.test.jsx

```diff
@@ -11,59 +11,73 @@ describe('<FeatureSelect />', () => {
     selectedFeature: FeatureSelect.featureTypes.PASSAGES.value,
     isFetchingResults: false,
   };
+  const propsWithRelevancy = Object.assign({}, props, {
+    selectedFeature: FeatureSelect.featureTypes.TRAINED.value,
+  });
+
+  let passagesTab;
+  let relevancyTab;
+  let highlightedFeature;
+
+  function renderWithProps(propSet) {
+    wrapper = shallow(<FeatureSelect {...propSet} />);
+    passagesTab = wrapper.find('.feature_select--list_button').at(0);
+    relevancyTab = wrapper.find('.feature_select--list_button').at(1);
+    highlightedFeature = wrapper.find('.feature_select--list_button--active');
+  }
 
   function getButtonClickEvent(button) {
     return { target: { value: button.props().value } };
   }
 
+  function pressTab(featureWrapper, tabNumber) {
```

Comment:
what if instead of passing in a `tabNumber` you passed in the named `let` you defined above? i.e. pass in `passagesTab` or `relevancyTab`